tests/Microsoft.Azure.StackExchangeRedis.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;net6.0</TargetFrameworks>
```

Comment:
To be consistent with the main project, should it be "netstandard2.0;net6.0"?

Reply:
No need for `netstandard2.0` because this is just a test project that won't be referenced by other projects. But it does make sense to retain .NET Framework 4.7.2 in case someone wants to run tests on a machine without .NET Core